runtime/syntax/nftables.yaml

```diff
@@ -0,0 +1,20 @@
+filetype: nftables
+
+detect:
+    filename: "nftables.conf$"
+    header: "^(#!.*/(env +)?nft( |$)|flush +ruleset)"
+    
+rules:
+    - type: "\\b(chain|counter|map|rule|ruleset|set|table)\\b"
+    - type: "\\b(ether|icmp|icmpv6|icmpx|inet|ip|ip6|ipv4|ipv6|tcp|udp)\\b"
+    - special: "\\b(elements|hook|policy|priority|type)\\b"
+    - identifier: "\\b(ct|iif|iifname|meta|oif|oifname|th)\\b"
+    - statement: "\\b(accept|drop|goto|jump|log|masquerade|reject)\\b"
+    - preproc: "\\b(add|define|flush|include|delete)\\b"
+    - symbol: "[-=/:;,@]"
+    - symbol.operator: "[<>.&|^!]|\\b(and|ge|gt|le|lt|or|xor)\\b"
```

Comment:
The `!=` isn't properly handled as unequal-operator. I'd assume something similar to:
https://github.com/zyedidia/micro/blob/8c0e0fa2ed38ab29bbbaf25818a030f616932af9/runtime/syntax/c.yaml#L23

Currently the `=` is tracked in line 14.